pkg/api/v20240812preview/openshiftcluster_validatestatic.go

```diff
@@ -469,7 +470,14 @@ func (sv openShiftClusterStaticValidator) validatePlatformWorkloadIdentityProfil
 	}
 
 	// Validate the PlatformWorkloadIdentities
+	foundIdentityResourceIDs := []string{}
```

Comment:
Nit: This could be a map instead of a slice so that we can check for duplicates in linear time instead of quadratic time, but pragmatically speaking it doesn't matter given the small number of platform workload identities.